docs/pages/reference/backends.mdx

```diff
@@ -620,6 +620,182 @@ Config option priority is applied in the following order:
   are not supported in certain regions or environments or are only supported in GovCloud.
 </Admonition>
 
+## Athena
+
+If you are running Teleport on AWS, you can use
+[Athena](https://aws.amazon.com/athena/) based audit log system that uses
+[Parquet files](https://parquet.apache.org/) stored on
+[S3](https://aws.amazon.com/s3/) as a storage back-end to achieve High
+Availability. Athena backend supports only one type of Teleport data:
+
+- Audit log events
+
+The athena audit logs backend provides better scalability and searchability in
+comparison to DynamoDB.
+
+Athena audit logs are eventually consistent, this means that events won't be
+immediately displayed in UI. It can take up to 1 minute until they are
+visible (depending on `batchMaxInterval` setting and events load).
+
+### Infrastructure set up
+
+Teleport uses AWS SNS and SQS subscribed to topic, for publishing events.
+One of Auth instances read events in batches from SQS and converts them into
+Parquet format and sends to S3.
+During query Athena engine is use to search for events on S3, via Glue table
+which contains metadata.
+
+Infrastructure can be set up using following terraform script.
+
+<Details title="Terraform script" opened={false}>
+```
+Add here terraform examples when they are merged.
+```
+</Details>
+
+### Configuring the Athena audit log backend
+
+To configure Teleport to use Athena:
+
+- Make sure you are using **Teleport version 14.0.0** or newer.
+- Prepare infrastructure
+- Use Athena URL insides `audit_events_uri` array.
+
+```yaml
+teleport:
+  storage:
+    # This setting configures Teleport to send the audit events to three places:
+    # To keep a copy in Athena, a copy on a local filesystem, and also output the events to stdout.
+    # More details about full athena url are shown below.
+    audit_events_uri:  ['athena://database.table?params', 'file:///var/lib/teleport/audit/events', 'stdout://']
+```
+
```

Comment:
We should introduce the URL here for readers who haven't seen the YAML comments above.

Reply:
The example URL appears after the configuration example with no context, so it's not immediately clear why it's there. I'll add a suggestion.